### ISSUE TITLE
Document protecting path with roles

### ISSUE THREAD

**Issue by ipeychev** [wedeploy/wedeploy.com#84]
<a href="https://github.com/ipeychev"><img src="https://avatars0.githubusercontent.com/u/78014?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [ipeychev](https://github.com/ipeychev)**
_Tuesday Jan 17, 2017 at 21:40 GMT_
_Originally opened as https://github.com/wedeploy/docs/issues/69_

----

Protecting given path with roles have to be added to the documentation. Here is an example:
```
{
  "data": true,
  "path": "/",
  "auth": {
    "scopes": ["admin"]
  }
}
```

**Comment by jonnilundy** [comment]
@ipeychev Could you share a simple demo or snippet that shows this so I can write the documentation. :) Thanks!

I'm also assuming that this should also cover how to add how to set user roles.

**Comment by ipeychev** [comment]
Hey @jonnilundy,

Unfortunately, my knowledge about this is not enough to write a precise documentation. Lais (or Eduardo?) told me the above in January and that is actually everything I know about these roles :(
The maitainer of Engine would be the best person for that, but since Lais is not here, that is currently @eduardolundgren and I'm not sure how busy he is :/ Probably we will have to read the sources/tests of Engine and extract the knowledge.

**Comment by jonnilundy** [comment]
This might be something that @igr could tackle? 😉

**Comment by jonnilundy** [comment]
Closed by #443
